.github/workflows/Java_CI.yaml

```diff
@@ -0,0 +1,16 @@
+name: Java CI
+
+on:
+  push:
+    branches: [ Feature/wrims-core ]
+
+jobs:
+  build:
+
+    runs-on: adoptopenjdk:8-jre-hotspot
```

Comment:
This should specify a version of the container

Reply:
actually this needs to be one of the GitHub runners: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners